uwblueprint/project-read-backend
# safe delete for students

### Notion ticket link



[Soft delete for students](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=c58b884b185841528522f5748684b3b4)



### Implementation description

- Used safe delete package
- By default students are now soft deleted. There is a date time field called deleted on the student model which indicates when a field was marked as deleted. They are not actually deleted from the db.
- we can undelete fields via Django admin



### Steps to test

1. Go to `http://localhost:8000/admin/registration/student/` and delete a student
2. Check that that student is highlighted as deleted
3. Try undeleting it from admin (select and undelete from admin actions)



### What should reviewers focus on?

- correctness

### Checklist
